benchmarks.yaml

```diff
@@ -200,4 +200,12 @@
   server: HonoJS
   run:
     cmd: bun
-    args: ["index.js"]
\ No newline at end of file
+    args: ["index.js"]
+- id: tailcall
+  name: tailcall
+  url: https://github.com/tailcallhq/tailcall
+  lang: Rust
```

Comment:
No Code

Reply:
For `lang`?

Reply:
Yes. Because the dev is actually not writing any rust code right?

Reply:
Yes that's correct. But not sure what does `lang` indicates here...implementation language or in which client will be using it.

Reply:
Readme.md files indicate `lang` as server language here. So Rust is correct

Reply:
@tusharmath Done!

Reply:
@sc231997  This is why you must at least understand before blind copy-pasting someone's code.